course/views.py

```diff
@@ -59,12 +61,14 @@ def get_common_objects(self):
         
         self.all_instances = all_instances
         self.my_instances = my_instances
+        self.is_internal = is_internal
```

Comment:
I'm not sure, but this could be seen as `INTERNAL` vs. `EXTERNAL` user thing and not as `internally stored and logged in user`. Could the variable name be something else?

Reply:
Hmm.. I tried to keep it short and describe that the user is internal (thus can have my courses). But what would you suggest instead?

Reply:
I am gonna change it to "is_logged_in". That describes that the user exists and is authenticated, thus can have my_courses